### PR TITLE
using the service through the API gateway requires an apikey

### PR DESCRIPTION
**Pull request by stefanpante**

### CODE REVIEW

**Comment by vademo** [comment]
Your implementation looks correct but Please fix the following  tasks

- [ ] Update the version with a minor or major release. 
- [ ] Add changes to changelog. 
- [ ] Fix the test that fails `npm run test`
- [ ] Update the documentation with the added parameter (readme.md).

**Comment by vademo** [comment]
@stefanpante You can make the decision. Theoretically it would be a Major because of the changes but if you prefer a Minor thats fine by me in this case.

File: src/chatbot/service.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { ServiceConfig, ChatbotMessage } from './types';
-import ChatMessager from './helpers/Chatmessager';
+import ChatMessager from './helpers/ChatMessager';
```
Comment (vademo) [inline src/chatbot/service.ts]: 👍

File: tests/integration/error.test.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ const mockedError = {
 
 describe('POST /api/chatbot`', () => {
   describe('Test Call', () => {
-    it('Expect The server to respond with te error given from the engine', async () => {
+    it('Expect The server to respond with the error given from the engine', async () => {
```
Comment (vademo) [inline tests/integration/error.test.ts]: 👍

File: tests/integration/messageInjection.test.ts [hunk 3]
```diff
@@ -163,19 +163,10 @@ describe('POST /api/chatbot/inject`', () => {
         },
         method: 'post',
         params: {
-          access_token: undefined,
+          access_token: "fake!",
```
Comment (vademo) [inline tests/integration/messageInjection.test.ts]: Single quote's please

**Comment by vademo** [comment]
> @stefanpante You can make the decision. Theoretically it would be a Major because of the changes but if you prefer a Minor thats fine by me in this case.

@stefanpante @TriangleJuice suggests it's a patch because the previous version didn't work as intended.

**Comment by stefanpante** [comment]
Ok, I'll update the changelog.

**Pull request merged**